_posts/2020-06-15-3D-Printing-During-Times-Of-Corona.md

```diff
@@ -0,0 +1,275 @@
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/shield1.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 49%; display: inline-block;">
+    </a>
+    <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/shield2.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Assembled face shield">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/shield2.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 49%; display: inline-block;">
+    </a>
+</div>
+<br/>
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.viewstl.com/?embedded&url=https://ordina-jworks.github.io/img/meeseeks/lid.stl" class="image fit" style="margin:0px auto; width: 100%; height: 350px; display: inline-block;"></iframe>
+</div>
+
+## Experimenting with TPU
+
+One of the disadvantages of regular materials like [PLA](https://en.wikipedia.org/wiki/Polylactic_acid){:target="_blank"}, [ABS](https://en.wikipedia.org/wiki/Acrylonitrile_butadiene_styrene){:target="_blank"} or [PETG](https://en.wikipedia.org/wiki/Polyethylene_terephthalate){:target="_blank"} is that they are fairly rigid, they do net flex very much or at all.
+When printing masks or devices that need to be worn on the head some form of flexibility is advisable to make the printed object conform to the shape of the head/face.
+
+Enter [TPU or Thermoplastic Polyurethane](https://en.wikipedia.org/wiki/Thermoplastic_polyurethane){:target="_blank"}.
+This material is flexible and can also be 3D printed.
+This makes the material suitable for use in designs that need to be able to conform to the contours of a human face.
+
+<div style="text-align: center; margin: 0px auto;">
+    <video width="270" height="480" autoplay muted loop>
+        <source src="/img/2020-06-15-3D-Printing-during-corona/IMG_0440.MOV" type="video/mp4">
+    </video>
+</div>
+<br/>
+
+The flexibility of the material is denoted on a hardness scale: The Shore hardness scale.
+
+<img alt="Shore hardness scale" src="{{ '/img/2020-06-15-3D-Printing-during-corona/shore-hardness.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 600px;">
+
+This scale has three scales:
+
+- Scale 00
+- Scale A: Soft rubber types
+- Scale D: Hard rubber types
+
+As you can see these scales do somewhat overlap.
+For 3D printing the harder the TPU material is, the easier it is to print.
+The lower the A or D scale value of the TPU material, the lower the printing speed and the shorter the retraction distance/speed has to be!
+
+TPU is harder to print than regular PLA or ABS.
+It is recommended to print TPU on a direct extruder at a lower than normal printing speed.
+However it can also be printed on bowden fed extruders, but the printing speed needs to be lowered even further to prevent the filament from spooling up or clogging in the bowden tube.
```

Comment:
```suggestion
However, it can also be printed on bowden fed extruders, but the printing speed needs to be lowered even further to prevent the filament from spooling up or clogging in the bowden tube.
```